content/get-started/index.md

```diff
@@ -0,0 +1,97 @@
+
+## **Galaxy Help & Bug Reports**
+
+<img class="img-fluid float-right" src="/images/undraw-illustrations/fixing-bugs.svg" style="width:200px;height:100px;" alt="community events" />
+
+Have a bug  <i style ="font-size:small;" class="fa fa-solid fa-bug"></i> or an error you haven't been able to solve? Do you have a question that you haven't found already answered in the FAQs? 
+
+Check our [**issues dedicated page**](/issues/) for known issues, problems and tips. Galaxy uses [**GitHub issues**](https://github.com/galaxyproject/galaxy/issues) for tracking down bugs and feature requests, **please make sure you search the list for duplicates before creating a new issue**.
+
+If you couldn't find a solution  reach out to the [**Galaxy Help Forum**](https://help.galaxyproject.org/) as *the* place to ask your questions related to Galaxy use or  [**create an issue**](https://docs.github.com/es/issues/tracking-your-work-with-issues/creating-an-issue) yourself. Either way we are more than happy to help you out!
+
+## **Mailing lists**
+If you haven't used mailing list before they are simply a list of addresses to which the same information is being sent, just another way to communicate and interact with the Community. See them [**here**](/mailing-lists/) and remember [**don't be afraid to ask questions**](mailing-lists#new-to-mailing-lists)
+
+
+# Step 3:  Your place within the Galaxy Community
+
+<img class="img-fluid float-right" src="/images/undraw-illustrations/connected-people.svg" style="width:200px;height:100px;" alt="community events" />
+
+Still wonder what the Galaxy Community is about?
+You already know it, you are already part of it! But let’s dive in the complexity of the [Galaxy Communities](/community/)!
+
+## **Regional Galaxy Communities**
+From India to Australia, Japan to France and many more countries [**Regional Communities**](/community/#regional-communities) are a big part of Galaxy. Find ways to contact them, platforms and lots of other useful links.
+
+## **Scientific Galaxy Communities**
+Another way communities are formed is by domains such as Proteomics,Transcriptomics,Ecology and Genome annotation to name a few. Most have set up their own servers and mailing list, look for them [**here**](/community/#regional-communities)
+
+## **Set up your own Community**
+What if you can't find yourself in these lists of regional and/or scientific communities? Galaxy welcomes new communities, please see [**here**](/get-started/new-leads).
+
+# Step 4: Take part and contribute to Galaxy
+
+[Here's](https://galaxyproject.org/community/contributing/) our guide on how to start contributing and there are many ways to do so: coding,writing and creating tutorials, answering questions in Gitter or the Help Forum and many more.
```

Comment:
a space is missing between coding,writing --> coding, writing

I would also add a link to gitter and agin the the help forum.